docs/source/write_first_app.rst

```diff
@@ -170,7 +170,7 @@ Open a new terminal, and navigate to the ``application-javascript`` folder.
 
 .. code:: bash
 
-  cd asset-transfer-basic/application-javascript
+  cd ../asset-transfer-basic/application-javascript
```

Comment:
Actually it says to open a new terminal, so we may want to say
`cd fabric-samples/asset-transfer-basic/application-javascript`
to be consistent with the earlier guidance.